partner_create_by_vat/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner Create by VAT
+=====================
+
+This module allows you to create the partners (companies) based on their
+VAT number. It will complete the name, address of the partner from
```

Comment:
Name and address of the partner will automatically be completed via VIES Webservice.